### ISSUE TITLE
ignore-cols for mlnet not working as it includes all other input columns to features

### ISSUE THREAD

**Issue by cyberkoolman** [dotnet/machinelearning-modelbuilder#2485]
**System Information (please complete the following information):**
 - OS & Version: Windows 10 & Windows 11
 - ML.NET Version: ML.NET v2.0.1
 - .NET Version: .NET 7.0

**Describe the bug**
Using the mlnet cli on wikiDetoxAnnotated40kRows.tsv which has the columns of Label, rev_id, comment, year, logged_in, ns, sample, split, wanted to pick only **comment** as the input feature.  Currently, through CLI, have to use --ignore-cols parameter and resulted the training pipeline to include all the columns to feature.  I had to resort to VS ModelBuilder for go around this issue for now.

**To Reproduce**
Execute cli as below
`mlnet classification --dataset .\wikiDetoxAnnotated40kRows.tsv --label-col Label --has-header true --ignore-cols rev_id year logged_in ns sample split --name Sentiment`

**Expected behavior**
I would expect the features only contain "comment" field

**Screenshots, Code, Sample Projects**
Resulting training pipeline as below
![image](https://user-images.githubusercontent.com/5167152/218235616-a709eb17-a060-409a-8f2e-588696feb703.png)

**Additional context**
1. Shouldn't this be easier if providing the option to choose only relevant column, instead of long list of ignore-columns.
2. Documentation says ignore-columns, not matching to ignore-cols.  [https://learn.microsoft.com/en-us/dotnet/machine-learning/reference/ml-net-cli-reference#ignore-columns]

**Comment by LittleLittleCloud** [comment]
@KeithT @cyberkoolman Thanks for reporting that, we are looking at it now.

@KeithT Can you update your model builder to latest version and see if the issue is still there?

**Comment by KeithT** [comment]
@LittleLittleCloud I updated to 17.14.4.2312404 and the ignore columns does now work. (Thanks). It still falls over when it's finished training and tries to write code though, as I reported here: 
https://github.com/dotnet/machinelearning-modelbuilder/issues/2541
I've posted a screenshot again below.
![Screenshot 2023-03-18 083836](https://user-images.githubusercontent.com/1142989/226092249-a029f4af-6bf5-4b26-8869-188ec01e29bd.jpg)
